.github/CODEOWNERS

```diff
@@ -26,15 +26,16 @@ testing/internal-testing/                   @gradle/bt-developer-productivity
 testing/precondition-tester                 @gradle/bt-developer-productivity
 
 # Release coordination
-packaging/core-platform/                @gradle/bt-product-operations
-packaging/distributions-dependencies/   @gradle/bt-product-operations
-packaging/distributions-full/           @gradle/bt-product-operations
-packaging/public-api                    @gradle/bt-product-operations
-testing/performance/                    @gradle/bt-developer-productivity
-testing/smoke-test/                     @gradle/bt-product-operations
-testing/soak/                           @gradle/bt-developer-productivity
-testing/distributions-integ-tests       @gradle/bt-developer-productivity
-testing/public-api-tests                @gradle/bt-developer-productivity
+packaging/core-platform/                        @gradle/bt-product-operations
+packaging/distributions-dependencies/           @gradle/bt-product-operations
+packaging/distributions-full/                   @gradle/bt-product-operations
+packaging/public-api                            @gradle/bt-product-operations
+testing/performance/                            @gradle/bt-developer-productivity
+testing/smoke-test/                             @gradle/bt-product-operations
+testing/smoke-test/src/**/*IsolatedProjects*    @gradle/bt-isolated-projects-project @gradle/bt-product-operations
```

Comment:
Added this line

Reply:
💅 I'd follow the pattern of putting the main owners first, and more specific later, like in other places in the file. This means `@gradle/bt-product-operations` would go first.